Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -72,7 +72,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        $hasPath = isset($params['path']);
+        $hasPath = !empty($params['path']);
```

Comment:
Could the same not be archived with "isset($params['path']) && $params['path'] !== ''"  ?

That way we can avoid empty() :)